README.md

```diff
@@ -1,10 +1,15 @@
 Microsoft Authentication Library for iOS and macOS
 =====================================
 
-| [Get Started](https://learn.microsoft.com/en-us/azure/active-directory/develop/mobile-app-quickstart?pivots=devlang-ios) | [iOS Sample Code](https://github.com/Azure-Samples/active-directory-ios-swift-native-v2) | [macOS Sample Code](https://github.com/Azure-Samples/active-directory-macOS-swift-native-v2) | [<u>B2C Sample Code</u>](https://github.com/Azure-Samples/active-directory-b2c-ios-swift-native-msal) | [Library reference](https://azuread.github.io/microsoft-authentication-library-for-objc/) | [Support](https://github.com/AzureAD/microsoft-authentication-library-for-objc/blob/dev/README.md#community-help-and-support) | [Feedback](https://forms.office.com/r/xuBV0CzEih)
-| --- | --- | --- | --- | --- | --- | --- | 
 
-The MSAL library for iOS and macOS gives your app the ability to begin using the [Microsoft Identity platform](https://aka.ms/aaddev) by supporting [Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) and [Microsoft Accounts](https://account.microsoft.com) in a converged experience using industry standard OAuth2 and OpenID Connect. The library also supports [Azure AD B2C](https://azure.microsoft.com/services/active-directory-b2c/) for those using our hosted identity management service.
+| Documentation                  | Sample Code               | Library Reference | Support | Feedback|
+|-------------------------------|---------------------------|-------------------|---------|----------|
+| [MSAL iOS and macOS documentation](https://learn.microsoft.com/en-us/entra/msal/objc/) <br/> [Workforce apps tutorials](https://learn.microsoft.com/en-us/entra/identity-platform/tutorial-v2-ios)<br/> Customer apps tutorials <br/>  • Browser-delegated <br/>• Native Authentication | [Workforce samples](https://learn.microsoft.com/en-us/entra/identity-platform/sample-v2-code?tabs=framework#ios) <br/> [Customer samples](https://learn.microsoft.com/en-us/entra/identity-platform/sample-v2-code?tabs=apptype#mobile)          | [ MSAL iOS and macOS reference](https://azuread.github.io/microsoft-authentication-library-for-objc/)              | [Support](https://github.com/AzureAD/microsoft-authentication-library-for-objc/blob/dev/README.md#community-help-and-support)    | [Feedback](https://forms.office.com/r/xuBV0CzEih) |
```

Comment:
Because the links are one under the other they aren't clear that they 2 different links.
There should be a way to differentiate them
<img width="490" alt="Screenshot 2024-03-04 at 13 15 05" src="https://github.com/AzureAD/microsoft-authentication-library-for-objc/assets/111577419/8ad0b997-b6fe-495a-af3e-7062f72d9663">
Not sure how the name on Github "Workforce apps tutorials" connects to the name in the link "Tutorial: Sign in users and call Microsoft Graph from an iOS or macOS app" - https://learn.microsoft.com/en-us/entra/identity-platform/tutorial-v2-ios
Same for the links under "Workforce samples" which leads to https://learn.microsoft.com/en-us/entra/identity-platform/sample-v2-code?tabs=framework#ios and the  MSAL iOS and macOS reference which leads to the Mobile - https://learn.microsoft.com/en-us/entra/identity-platform/sample-v2-code?tabs=apptype#mobile

Reply:
Thanks @spetrescu84 for an early review here.

For the documentation updates and links:

- We point to the official MSAL ObjC docs on MS Learn - https://learn.microsoft.com/en-us/entra/msal/objc/
- Within the getting started section of the readme, we provide links to the different quickstarts/tutorials that devs can follow, based on their scenario and type of app they're building (workforce/customer)
- I've also removed much of the content that's duplicated in the official docs (configuration, initialize client app, and acquire tokens) and instead pointed to the guidance on MS Learn, i.e:
     * [Configure your project to use MSAL](https://learn.microsoft.com/en-us/entra/msal/objc/install-and-configure-msal#configuring-your-project-to-use-msal)
    * [Configure authority for different identities](https://learn.microsoft.com/en-us/entra/msal/objc/configure-authority)
    * [Configure redirect URIs](https://learn.microsoft.com/en-us/entra/msal/objc/redirect-uris-ios)
    * [Acquire tokens](https://learn.microsoft.com/en-us/entra/msal/objc/acquire-tokens)

As for the code samples, we also need to differentiate the between workforce and customer sample apps, that's why we have two links. Eventually, the links will point to the code samples browser, such as (https://learn.microsoft.com/en-us/samples/browse/?products=microsoft-authentication-library) I'm working to get all code samples for iOS and macOS onboarded there.